app/scraper.py

```diff
@@ -29,10 +31,11 @@ def frankenbert_rank(user_input: str) -> Tuple[str, str, Tuple[int, float]]:
 
 def deduplicate(new_data: List[Dict]) -> List[Dict]:
     """ Checks for duplicates and omits them """
-    old_data = load_data()
+
+    old_data = DB.load_data()
```

Comment:
Can we make this more efficient and just grab the `tweet_id`s instead of all columns? Since we are only using this to check against incoming IDs and appending new incoming data to a list, we won't need to have all the old data stored in the `old_data` variable. It may likely become more necessary, as the database is increasingly filled with thousands of rows, to lighten the load of this request a bit and keep things quick and snappy.